client_configuration.go

```diff
@@ -129,14 +164,29 @@ func validateClientConfiguration(clientConfig *ClientConfig) error {
 func validateLogLevel(clientConfig ClientConfig) error {
 	var logLevel = clientConfig.Common.LogLevel
 	if logLevel != "" {
-		_, error := toLogLevel(logLevel)
-		if error != nil {
-			return error
+		_, err := toLogLevel(logLevel)
+		if err != nil {
+			return err
 		}
 	}
 	return nil
 }
 
+func isCfgPermValid(filePath string) (bool, error) {
+	if runtime.GOOS != "windows" {
```

Comment:
actually for me it would look better:
```
if runtime.GOOS == "windows" {
  return true, nil
}
```

and then logic for linux/mac.

BTW do we have any test showing that this condition `runtime.GOOS != "windows"` is working well?

Reply:
Good idea, done.

We do not have such test, but this is a widely adopted way of checking the system in Go.